src/pages/BorrowerProfile/BorrowerProfile.vue

```diff
@@ -0,0 +1,35 @@
+<template>
```

Comment:
Please add `www-page` as the root element (e.g. https://github.com/kiva/ui/blob/master/src/pages/BorrowerProfile/fundedBorrowerProfile.vue#L2). That will add the header and footer to the page.

Reply:
Just added.